releases/v1.9.5.toml

```diff
@@ -0,0 +1,13 @@
+# The commit/tag of the release
+commit = "v1.9.5"
+project_name = "CRI-O"
+github_repo = "kubernetes-incubator/cri-o"
+previous = "v1.9.3"
```

Comment:
this should be v1.9.4 or the changes are wrong (cause they're counted and reported from 1.9.3 rather than 1.9.4)